perf/perf.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+#set variables
+CUR_DIR=$(pwd)
+BASE_DIR=`dirname $CUR_DIR`
+
+GOLD_STANDARD_PERF_FILE="$BASE_DIR/perf/gold_standard_bench.txt"
+LATEST_PERF_FILE="$BASE_DIR/perf/latest_bench.txt"
+TEMP_FILE="$BASE_DIR/perf/temp_bench.txt"
+
+BENCH_DIRS=($BASE_DIR/"internal/crypto")
```

Comment:
Why do you only consider `internal/crypto` for benchmarking?

Reply:
I was only testing on internal/crypto. This script would be updated when I write more tests for those packages.